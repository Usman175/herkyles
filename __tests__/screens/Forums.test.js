import React from "react";
import Forums from "../../app/screens/Forums";

import renderer from "react-test-renderer";

describe ("Forums", () => {
  it("renders without crashing", () => {
    const rendered = renderer.create(<Forums />).toJSON();
    expect(rendered).toBeTruthy();
    expect(rendered).toMatchSnapshot();
  });
});