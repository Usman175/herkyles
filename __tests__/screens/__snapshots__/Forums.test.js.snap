// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Forums renders without crashing 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hitSlop={undefined}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={undefined}
  testID={undefined}
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 15,
        "textAlign": "center",
      }
    }
  >
    Forums
  </Text>
</View>
`;
